include/ListView/ListViewSubPanel.php

```diff
@@ -539,7 +539,9 @@ public function processListNavigation($smartyTemplateSection, $html_varName, $cu
             global $currentModule;
             global $app_strings;
 
-            $start_record = $current_offset + 1;
+            if(!isset($current_offset) || empty($current_offset))                                                                    
```

Comment:
Please stick to the format of the [Coding Standards](https://suitecrm.com/wiki/index.php/Coding_Standards#Brace_Style)

So set it as:
if(!isset($current_offset) || empty($current_offset)){
  $current_offset=0;
}